mne/io/eeglab/tests/test_eeglab.py

```diff
@@ -129,6 +129,24 @@ def test_io_set_raw_more(tmpdir):
                     overlap_fname.replace('.set', '.fdt'))
     read_raw_eeglab(input_fname=overlap_fname, preload=True)
 
+    # test reading file with empty event durations
+    empty_dur_fname = op.join(tmpdir, 'test_empty_durations.set')
+    evnts = deepcopy(eeg.event)
+    for ev in evnts:
+        ev.duration = np.array([], dtype='float')
+
+    io.savemat(empty_dur_fname,
+               {'EEG': {'trials': eeg.trials, 'srate': eeg.srate,
+                        'nbchan': eeg.nbchan,
+                        'data': 'test_negative_latency.fdt',
+                        'epoch': eeg.epoch, 'event': evnts,
+                        'chanlocs': eeg.chanlocs, 'pnts': eeg.pnts}},
+               appendmat=False, oned_as='row')
+    shutil.copyfile(op.join(base_dir, 'test_raw.fdt'),
+                    empty_dur_fname.replace('.set', '.fdt'))
+    raw = read_raw_eeglab(input_fname=empty_dur_fname, preload=True)
+    assert (raw.annotations.duration == 0).all()
```

Comment:
Shouldn't some or all of these been `np.nan` based on the conditional in `eeglab.py` and the `ev.duration = np.array([], dtype='float')` above?

Reply:
No, the eeglab reader turns `np.nan` somewhere later into zeros. I can use zeros instead of nans if you think this would be better. But we may also decide to expose the nan durations in annotations at some time in the future.

Reply:
Makes sense!